tests/test_basics.py

```diff
@@ -438,6 +438,35 @@ def dummy(needed):
         with self.assertRaises(TypeError):
             dummy()
 
+    def _working_normally(self):
+        with patch('logging.Logger.log') as logger:
+            res = aaa()
+            self.assertEqual(res, 'this')
+            self.assertEqual(logger.call_count, 2)
+            (alert, logged_msg), extras = logger.call_args_list[0]
+            self.assertTrue(logged_msg.startswith('*Called'))
+            (alert, logged_msg), extras = logger.call_args_list[-1]
+            self.assertTrue(logged_msg.startswith('*Returned'))
+        return True
+
+    def _not_logging(self):
+        with patch('logging.Logger.log') as logger:
+            res = aaa()
+            self.assertEqual(res, 'this')
+            logger.assert_not_called()
+        return True
+
+    def test_stop_and_start(self):
+        """
+        Check that the start and stop commands actually do something
+        """
+        loggo.start()
+        self.assertTrue(self._working_normally())
```

Comment:
whats the purpose of these asserts, or the return values of `_working_normally` and `_not_logging`? Seems to me that True is always the return value and these asserts can never fail?

Reply:
is done